### PR TITLE
MSD-80376-Fix: Added SizeRestrictions_QUERYSTRING to the WAF excluded…

### PR DESCRIPTION
**Pull request by YD-DLUHC**
… rules

Fix for [MSD-80376](https://dluhcdigital.atlassian.net/browse/MSD-80376)

[MSD-80376]: https://dluhcdigital.atlassian.net/browse/MSD-80376?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by hugh-emerson — APPROVED** [review]
But you might want to split the list over mulitiple lines so the "# Orbeon triggers this rule" comment can be next to the `CrossSiteScripting_BODY` line.

The GenericLFI_BODY line was to allow some xlsx file upload I believe.

**Comment by hugh-emerson** [comment]
To be thorough, you should check you can reproduce the issue on staging, deploy the fix and then check it's resolved

**Comment by YD-DLUHC** [comment]
> But you might want to split the list over mulitiple lines so the "# Orbeon triggers this rule" comment can be next to the `CrossSiteScripting_BODY` line.
> 
> The GenericLFI_BODY line was to allow some xlsx file upload I believe.
> 
> Allowing long query parameters doesn't really need an explanation. But I guess you could leave a comment with the MSD ticket number.

Split rules over multiple lines and left a comment

**Comment by YD-DLUHC** [comment]
> To be thorough, you should check you can reproduce the issue on staging, deploy the fix and then check it's resolved

I've tested on stage pre and post release and it works now

File: terraform/modules/cloudfront_distributions/main.tf [hunk 1]
```diff
@@ -23,7 +23,10 @@ module "delta_website_waf" {
   log_group_suffix  = "delta-website-${var.environment}"
   per_ip_rate_limit = var.waf_per_ip_rate_limit
   # Orbeon triggers this rule
-  excluded_rules                 = ["CrossSiteScripting_BODY", "GenericLFI_BODY"]
+  excluded_rules = ["CrossSiteScripting_BODY",
+    "GenericLFI_BODY",
+    # To allow long query strings - fix for MSD-80376
+  "SizeRestrictions_QUERYSTRING"]
```
Comment (BenRamchandani) [inline terraform/modules/cloudfront_distributions/main.tf:29]: I think Hugh meant formatted like this, lgtm anyway

```suggestion
  excluded_rules = [
    "CrossSiteScripting_BODY",     # Sometimes blocks File uploads
    "GenericLFI_BODY",             # Sometimes blocks Excel uploads
   "SizeRestrictions_QUERYSTRING", # To allow long query strings - fix for MSD-80376
 ]
```

**Pull request merged**